content/ru/observability.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Observability
+status: Completed
+category: concept
+tags: ["property", "", ""]
+---
+
+Наблюдаемость — это свойство системы, которое показывает, насколько легко можно получить полезную информацию о её состоянии. 
```

Comment:
It might be excessive(?), but personally, I'd prefer to keep the original term here as well. If we change the formal title above, then it's necessary.
```suggestion
Наблюдаемость (observability) — это свойство системы, которое показывает, насколько легко можно получить полезную информацию о её состоянии. 
```